dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/Status.java

```diff
@@ -33,6 +33,7 @@
 public enum Status
 {
     OK,
+    SUCCESS,
```

Comment:
How is this different from `OK`? Wasn't that the _success_ state already?

Reply:
Good question, so I am getting `SUCCESS` from the data value set API in 2.37 and `OK` in 2.38.

So when targeting older DHIS 2 version for data exchange, it breaks. I can add a new enum to the data exchange project to contain this if that is better.

Reply:
I see, I think this is due to changing the type of report we return or report vs. web message response. Maybe the extra enum is the best compromise to keep this odd exception local.

Reply:
Makes sense. Fixed.